src/__tests__/note.test.ts

```diff
@@ -46,6 +46,21 @@ describe('comment generation', () => {
     expect(comment).toEqual(expect.stringContaining(constants.NOTES_LEAD));
     expect(comment).toEqual(expect.stringContaining('> line one\n> line two'));
   });
+
+  it('can handle different multiline note formatting', () => {
+    const note = noteUtils.findNoteInPRBody(prBodyWithMultilineNotes);
+    const comment = noteUtils.createPRCommentFromNotes(note);
+    expect(comment).toEqual(expect.stringContaining(constants.NOTES_LEAD));
+
+    const expected = `
+> * Fixes the following issues for frameless when maximized on Windows:
+> * fix unreachable task bar when auto hidden with position top
+> * fix 1px extending to secondary monitor
+> * fix 1px overflowing into taskbar at certain resolutions
+> * fix white line on top of window under 4k resolutions`;
```

Comment:
IMO this is kind of crappy formatting for the final actual release notes, as the 2nd to 5th list items are at the same level as the 1st, not beneath it.

I think this is poor formatting in the original PR. Ideally we'd have either:

```
* Fixes the following issues...:
  * Unreachable task bar ...
  * 1px extending...
  ⋮
```
or, and I think I prefer this one, each line should be readable on its own, and the top sentence should have been excluded from the original PR:
```
* Fixed unreachable task bar...
* Fixed 1px extending...
⋮
```